src/coreclr/vm/jitinterface.cpp

```diff
@@ -7680,20 +7680,16 @@ CEEInfo::getMethodInfo(
     JIT_TO_EE_TRANSITION();
 
     MethodDesc* ftn = GetMethod(ftnHnd);
-    if (ftn->IsDynamicMethod())
```

Comment:
Why is it ok to delete this?

Reply:
Right, `IsDynamicMethod()` does not satisfy `sIL()`, so `else if (ftn->IsIL())` below needs to include IsDynamicMethod explicitly.